etstool.py

```diff
@@ -374,5 +376,37 @@ def execute(commands, parameters):
             sys.exit(1)
 
 
+def locate_edm():
+    """
+    Locate an EDM executable, and raise a click exception if not found.
+
+    Returns
+    -------
+    edm : str
+        Path to the EDM executable to use.
+    """
+    if sys.platform == "win32":
+        cmd = ["where", "edm"]
+    else:
+        cmd = ["which", "edm"]
```

Comment:
IIRC, the reason why can't use `shutil.where` is because we want to continue supporting python 2 environments as bootstrap environments. Is that correct?

And if it is, do we want to continue supporting py2 envs as bootstrap envs?

Reply:
> And if it is, do we want to continue supporting py2 envs as bootstrap envs?

Yes, until we drop support for Python 2 altogether in Q1 2020.